byoeb-v1/byoeb/byoeb/apis/background_jobs.py

```
@@ -30,20 +37,143 @@
 account_url = "https://khushibabyashastorage.blob.core.windows.net"
 container_name = "ashacontainer"
 
-# Linux/Production server commands (using exec)
-background_jobs = [
-    f"*/30 * * * * exec python3 {jobs_path}/consensus/respond_with_consensus.py; exit",
-    f"00 8-20 * * * exec python3 {jobs_path}/consensus/send_query_to_expert.py; exit",
-    f"0 12 * * FRI exec python3 {jobs_path}/message_leaderboard/leaderboard.py; exit",
+# Job configuration with proper cron expressions and function imports
+JOB_CONFIGURATIONS = [
+    {
+        "id": "consensus_responder",
+        "name": "Respond with Consensus",
+        "cron": "*/30 * * * *",  # Every 30 minutes
+        "function": "main",
+        "module": "byoeb.background_jobs.consensus.respond_with_consensus",
```

Invalid job functions go undetected until the scheduler tries to execute them.

In `setup_scheduled_jobs()`, we must validate whether `"module"` exists (and `"function"` too), so that ASHAbot [fails fast](https://en.wikipedia.org/wiki/Fail-fast_system) early during app lifecycle.

Alternatively (and this is probably better), we can by-design ensure job functions exist by resolving these symbols at import-time (instead of at runtime).
```diff
  # ModuleNotFoundError / ImportError catches this early @@
+ from byoeb.background_jobs.consensus.respond_with_consensus import main as respond_with_consensus

JOB_CONFIGURATIONS = [
    {
        "id": "consensus_responder",
        "name": "Respond with Consensus",
        "cron": "*/30 * * * *",
-       "function": "main",
-       "module": "byoeb.background_jobs.consensus.respond_with_consensus",
+       "function": respond_with_consensus,
        "enabled": True
    }
]
```

---

Added in latest commit